src/utils/EventUtils.js

```diff
@@ -50,3 +51,35 @@ export function canEditContent(mxEvent) {
         mxEvent.getOriginalContent().msgtype === "m.text" &&
         mxEvent.getSender() === MatrixClientPeg.get().getUserId();
 }
+
+export function findPreviousEditableEvent(room, fromEventId = undefined) {
```

Comment:
Making this and `findNextEditableEvent` be a single event is probably worthwhile, given the amount of duplication between them. Something like `findEditableEvent(room, forwards=true, fromEventId=undefined)` is better imo (using the flag to determine the start index and the `findIndex` function to use).

Reply:
Did this, not sure it's more readable, but it indeed has less duplication. Wdyt?

Reply:
Yea, that turned out less readable than I thought, but I think I'm more in favour of it because less duplication. It's also understandable code, which helps.